scripts/evm/create_go_bindings.sh

```diff
@@ -34,6 +37,11 @@ do
     
     # Build the go package
     abigen --abi $ABI_DIR/${value}.abi --pkg $value --type $value --bin $BIN_DIR/${value}.bin --out $GO_DIR/$value/$value.go
+    
+    # Capture build temp 
+    bytecode=`cat ./build/tmp/runtime/Bridge.bin`
+    variable="var RuntimeBytecode = '${bytecode}'"
```

Comment:
`${bytecode}` should be surrounded by double rather than single quotes.

Reply:
I'm actually trying to capture the value in quotes

Reply:
If that is the case, shouldn't this line be `\"${bytecode}\""` ?